chapter_notation/index.md

```diff
@@ -1,85 +1,94 @@
-* $\mathbf{X}$: A matrix
-* $\mathsf{X}$: A tensor
-* $\mathbf{I}$: An identity matrix
-* $x_i$, $[\mathbf{x}]_i$: The $i^\mathrm{th}$ element of vector $\mathbf{x}$
-* $x_{ij}$, $x_{i,j}$,$[\mathbf{X}]_{ij}$, $[\mathbf{X}]_{i,j}$: The element of matrix $\mathbf{X}$ at row $i$ and column $j$
+## Numerical Objects
+
+* $x$: a scalar
+* $\mathbf{x}$: a vector
+* $\mathbf{X}$: a matrix
+* $\mathsf{X}$: a general tensor
+* $\mathbf{I}$: an identity matrix---square, with $1$ on all diagonal entries and $0$ on all off-diagonals
+* $x_i$, $[\mathbf{x}]_i$: the $i^\mathrm{th}$ element of vector $\mathbf{x}$
+* $x_{ij}$, $x_{i,j}$,$[\mathbf{X}]_{ij}$, $[\mathbf{X}]_{i,j}$: the element of matrix $\mathbf{X}$ at row $i$ and column $j$.
 
 
 
 
 ## Set Theory
 
 
-* $\mathcal{X}$: A set
-* $\mathbb{Z}$: The set of integers
-* $\mathbb{Z}^+$: The set of positive integers
-* $\mathbb{R}$: The set of real numbers
-* $\mathbb{R}^n$: The set of $n$-dimensional vectors of real numbers
+* $\mathcal{X}$: a set
+* $\mathbb{Z}$: the set of integers
+* $\mathbb{Z}^+$: the set of positive integers
+* $\mathbb{R}$: the set of real numbers
+* $\mathbb{R}^n$: the set of $n$-dimensional vectors of real numbers
 * $\mathbb{R}^{a\times b}$: The set of matrices of real numbers with $a$ rows and $b$ columns
-* $|\mathcal{X}|$: Cardinality (number of elements) of set $\mathcal{X}$
-* $\mathcal{A}\cup\mathcal{B}$: Union of sets $\mathcal{A}$ and $\mathcal{B}$
-* $\mathcal{A}\cap\mathcal{B}$: Intersection of sets $\mathcal{A}$ and $\mathcal{B}$
-* $\mathcal{A}\setminus\mathcal{B}$: Subtraction of set $\mathcal{B}$ from set $\mathcal{A}$
+* $|\mathcal{X}|$: cardinality (number of elements) of set $\mathcal{X}$
+* $\mathcal{A}\cup\mathcal{B}$: union of sets $\mathcal{A}$ and $\mathcal{B}$
+* $\mathcal{A}\cap\mathcal{B}$: intersection of sets $\mathcal{A}$ and $\mathcal{B}$
+* $\mathcal{A}\setminus\mathcal{B}$: set subtraction of $\mathcal{B}$ from $\mathcal{A}$ (contains only those elements of $\mathcal{A} that do not belong to $\mathcal{B}$)
 
 
 ## Functions and Operators
 
 
-* $f(\cdot)$: A function
-* $\log(\cdot)$: The natural logarithm
-* $\exp(\cdot)$: The exponential function
-* $\mathbf{(\cdot)}^\top$: Transpose of a vector or a matrix
-* $\mathbf{X}^{-1}$: Inverse of matrix $\mathbf{X}$
+* $f(\cdot)$: a function
+* $\log(\cdot)$: the natural logarithm (base $e$)
+* $\log_2(\cdot)$: logarithm with base $2$
+* $\exp(\cdot)$: the exponential function
+* $\mathbf{1}(\cdot)$: the indicator function, evaluates to $1$ if the boolean argument is true and $0$ otherwise
+* $\mathbf{1}_{\mathcal{X}}(z)$: the set-membership indicator function, evaluates to $1$ if the element $z$ belongs to the set $\mathcal{X}$ and $0$ otherwise
+* $\mathbf{(\cdot)}^\top$: transpose of a vector or a matrix
+* $\mathbf{X}^{-1}$: inverse of matrix $\mathbf{X}$
 * $\odot$: Hadamard (elementwise) product
-* $[\cdot, \cdot]$: Concatenation
-* $\lvert \mathcal{X} \rvert$: Cardinality of set $\mathcal{X}$
```

Comment:
The set cardinality notation is widely used throughout the book so we can keep it

Reply:
its deleted because it was redundant (appeared in two places, both set notation and function notation)